src/metadata-stream.js

```diff
@@ -42,6 +42,7 @@
             break;
           }
         }
+        tag.data = tag.value;
```

Comment:
Could we add [deprecation warnings](https://github.com/videojs/videojs-contrib-hls/blob/72ece371988ef4f18ef1c6b38074a1120fdffbb5/src/videojs-hls.js#L38-L43) for the old property locations so we can clean them up before too long?